tests/auth_tests/nomigrations/nomigrations/settings.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+SECRET_KEY = 'django_auth_tests_secret_key'
+
+INSTALLED_APPS = [
+    'django.contrib.auth.apps.BaseAuthConfig',
+    'django.contrib.contenttypes',
+]
+
+MIGRATION_MODULES = {'auth': None}
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.environ['DJANGO_AUTH_TESTS_DB'],
```

Comment:
Maybe the settings file could be written dynamically to avoid the environment variable and repetition of DATABASES in the test?